DataFormats/METReco/interface/BeamHaloSummary.h

```diff
@@ -70,6 +73,12 @@ namespace reco {
     std::vector<int>& GetGlobaliPhiSuspects() { return GlobaliPhiSuspects ;}
     const std::vector<int>& GetGlobaliPhiSuspects() const { return GlobaliPhiSuspects ;}
 
+    std::vector<std::vector<std::pair<char, CaloTowerDetId> > >& GetProblematicStrips() { return ProblematicStrips ;}
```

Comment:
.. if you convert to CaloTowerIdsWithQuality, or just HaloTowerStrip for a more intuitive name, you can also put the hadEt value inside it to have one-stop access to the details /properties of the strip itself.